### PR TITLE
Fix UniLinks Plugin for Flutter 1.12.13

### PR DESCRIPTION
**Pull request by markathomas**

### CODE REVIEW

File: android/src/main/java/name/avioli/unilinks/UniLinksPlugin.java [hunk 1]
```diff
@@ -3,111 +3,139 @@
...
+
+    @Override
+    public boolean onNewIntent(Intent intent) {
+        this.handleIntent(context, intent);
+        return false;
+    }
+
+    @Override
+    public void onAttachedToActivity(ActivityPluginBinding activityPluginBinding) {
+        this.handleIntent(this.context, activityPluginBinding.getActivity().getIntent());
```
Comment (petermnt) [inline android/src/main/java/name/avioli/unilinks/UniLinksPlugin.java]: You seem to be missing the new intent listener here and in the reattach
`activityPluginBinding.addOnNewIntentListener(this);`

**Comment by markathomas** [comment]
Thanks for that, added.

**Comment by ened** [comment]
@avioli hi! This is a useful PR. Are you still maintaining the project?

**Comment by Zazo032** [comment]
@avioli friendly ping to check if this can be merged to the repo and published to pub

**Comment by TBG-FR** [comment]
Should we do something to integrate all these modifications and maintain/update the plugin ? A new repo maybe ? No news since October, and opening every fork to manually check the commits and judge their rightness is a hassle...

**Comment by markathomas** [comment]
I concur, I am using my branch in production.  I don't have the time currently to properly maintain this plugin but if you'd like to use my repo I'd be happy to add anyone willing to help maintain it to the repo.  A new plugin name and pub publish is probably the best way to go.

**Comment by TBG-FR** [comment]
Thanks a lot for your work @avioli !

**Comment by avioli** [comment]
Thanks to @markathomas

**Comment by markathomas** [comment]
yes
Regards,

Mark Thomas
spatialguru.net@gmail.com
205.529.9013

"Commit to the Lord whatever you do,
    and your plans will succeed." - Proverbs 16:3

**Pull request merged**